pkg/kubernetes/informers.go

```diff
@@ -101,6 +119,9 @@ func (im *InformerManager) AddConfigMapListener(add func(obj interface{}), updat
 		UpdateFunc: update,
 		DeleteFunc: remove,
 	})
+	stopCh := make(chan struct{})
```

Comment:
Dont you need to close this channel at some point?

Reply:
No, it is similar to shareInformerfactory's Start() method. If this channel is closed, I have observed that the caller receives a shutdown signal
```
time="2020-11-10T21:11:48Z" level=info msg="received signal; shutting down" signal=terminated
time="2020-11-10T21:11:48Z" level=info msg="gracefully stopped"
time="2020-11-10T21:11:48Z" level=info msg="removed sock file" path=/csi/csi.sock
time="2020-11-10T21:11:48Z" level=info msg="server stopped gracefully"```

Also, verified the Start() method in SharedInformerFactory:
Refer: https://github.com/kubernetes/client-go/blob/master/informers/factory.go#L134